.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: c
+sudo: false
```

Comment:
`sudo: false` was [recently deprecated](https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration) by travis, so this line has no effect. Can you remove it?